### PR TITLE
Implement parser logic

### LINKED ISSUE #91: Implement the EditGameCommandParser class and subsequent logic

### PR DESCRIPTION
**Pull request by Xczheng0105**
fixes #91

### CODE REVIEW

**Review by JJtan2002 — APPROVED** [review]
LGTM! Merging

File: src/main/java/seedu/address/model/game/SkillLevel.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  */
 public class SkillLevel {
 
-    private static final String MESSAGE_CONSTRAINTS =
+    public static final String MESSAGE_CONSTRAINTS =
             "Skill Level should not be blank";
```
Comment (flyingsalsa) [inline src/main/java/seedu/address/model/game/SkillLevel.java:12]: Good catch on the missing message_constraints

File: src/main/java/seedu/address/logic/commands/EditGameCommand.java [hunk 2]
```diff
@@ -73,7 +68,19 @@ public EditGameCommand(Index index, String gameName, EditGameDescriptor editGame
 
     @Override
     public CommandResult execute(Model model) throws CommandException {
-        throw new CommandException("This command should return an edited game");
+        requireNonNull(model);
+        List<Person> lastShownList = model.getFilteredPersonList();
+        if (index.getZeroBased() >= lastShownList.size()) {
```
Comment (flyingsalsa) [inline src/main/java/seedu/address/logic/commands/EditGameCommand.java:73]: Maybe an explanation for getZeroBased() would be good, the name isnt very self-explanatory

**Review by flyingsalsa — COMMENTED** [review]
LGTM!

**Pull request merged**